src/frontend/src/components/AboutFooter/AboutFooter.jsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { makeStyles, useTheme } from '@material-ui/core/styles';
 import { Box, Divider, Grid, Typography, useMediaQuery } from '@material-ui/core';
 import GitHubIcon from '@material-ui/icons/GitHub';
+// import useTelescopeContributor from '../../hooks/use-telescope-contributor';
```

Comment:
Modify this file only when you send the components to display it.
Perform these tests on your local machine, but don't ship it until you have the entire work. ( Don't send  imports commented code)

The main focus of this PR is the hooks. So keep the scope on it, send only the hooks.